addons/gamification/models/challenge.py

```diff
@@ -553,9 +553,7 @@ def report_progress(self, users=(), subset_goals=False):
                 partner_ids=challenge.mapped('user_ids.partner_id.id'),
                 subtype='mail.mt_comment',
                 notif_layout='mail.mail_notification_light',
-                notif_values={
-                        'model_description': challenge._description.lower(),
-                    }
+                model_description=challenge._description.lower()
```

Comment:
Actually normally model_description is not necessary as we are posting on a record: model description will come from model / res_id. This is valid for other post in this file I think.